### DISCUSSION TITLE
Binding to UserControl DirectProperty<AnotherUserControl>

### DISCUSSION THREAD

**Discussion by Sophisticated-IS** [AvaloniaUI/Avalonia#16361] [Q&A] [answered]
**Question:** Is it possible to bind to UserControl.AnotherControl property where "AnotherControl" is direct property of "User Control"?
**Problem:**  I would not like to copy the same DirectProperties to Main User Control in case to be able to bind to it
**Example:**
```
<p:CustomUserControl 
                   DirectPropUserControl.CustomProperty="{Binding ViewModelProperty, Mode=OneWay}">
</p:CustomUserControl>
```

**Comment by maxkatz6** [comment, marked as answer]
No, `CustomProperty` is owned by `DirectPropUserControl`, so `CustomUserControl` won't be able to have it set.

You either want to use AttachedProperty or register the same property on CustomUserControl but with `CustomProperty = DirectPropUserControl.CustomProperty.AddOwner()` instead of full syntax (we use it [in some places](https://github.com/AvaloniaUI/Avalonia/blob/8ea60fefe24b508a7de9fcadd28d67a25c8afb8a/src/Avalonia.Controls/ContextMenu.cs#L26-L27) as well)
